.circleci/config.yml

```diff
@@ -53,6 +50,7 @@ commands:
       - run:
           name: Run Tests 5.0
           command: |
+            apt-get install -y dotnet-sdk-5.0
```

Comment:
There's also a version 5 SDK image available, so you could remove this line and put this on line 12 above:

`- image: mcr.microsoft.com/dotnet/sdk:5.0`

Reply:
Since this passes, it's probably not necessary, just another option.